### PR TITLE
Add printer-friendly cheatsheet

### PR DESCRIPTION
**Pull request by jakubvalenta**
- Generate a cheatsheet that has black text on white background, which would use less toner when printed.
- Document how to generate this cheatsheet in README.
- Document how to generate PNGs from cheatsheet PDFs.
- Refactoring: Make `prelude-cheatsheet.pdf` a symlink to `modules/doc/prelude-cheetsheet.pdf`.

Resolves issue: #1162

cheatsheet-light-800

### CODE REVIEW

File: modules/doc/prelude-cheatsheet.tex [hunk 1]
```diff
@@ -12,15 +12,13 @@
 \usepackage{calc}
 \usepackage{tcolorbox}
 
-\pagecolor{black}
+\usepackage{style-dark}
```
Comment (bbatsov) [inline modules/doc/prelude-cheatsheet.tex]: Oh, I didn't notice this. Seems to me we can simply change the default.

Comment (jakubvalenta) [reply modules/doc/prelude-cheatsheet.tex]: So if I understand correctly, your suggestion is to only have one version of the cheatsheet - the one with black text on white background? I personally don't care about the original dark version, but I think it's nice if the users can choose between the light or dark version according to their liking.

Comment (bbatsov) [reply modules/doc/prelude-cheatsheet.tex]: Let's not overdo this and have just a single version. :-) 

I don't think it makes sense to make customizable something as basic as this, that's probably used by about 10 people. :-)

Comment (jakubvalenta) [reply modules/doc/prelude-cheatsheet.tex]: Ok, if we're not married to the dark version as you say, then I will make a new PR with just the colors adjusted. I'm closing this PR now. Thanks for your advice.

**Pull request closed without merging**